Corona-Warn-App/src/main/java/de/rki/coronawarnapp/util/di/ApplicationComponent.kt

```diff
@@ -88,6 +93,7 @@ import javax.inject.Singleton
         SerializationModule::class,
         WorkerBinder::class,
         StatisticsModule::class,
+        AnalyticsModule::class,
```

Comment:
it is already included in  `DataDonationModule`

Reply:
removed AnalyticsModule here